flaml/default/suggest.py

```diff
@@ -1,259 +1,260 @@
-            columns in the dataframe are assumed to be exogenous
-            variables (categorical or numeric).
-        y: A series of labels in shape n*1.
-        estimator_or_predictor: A str of the learner name or a dict of the learned config predictor.
-            "choose_xgb" means choosing between xgb_limitdepth and xgboost.
-            If a dict, it contains:
-            - "version": a str of the version number.
-            - "preprocessing": a dictionary containing:
-                * "center": a list of meta feature value offsets for normalization.
-                * "scale": a list of meta feature scales to normalize each dimension.
-            - "neighbors": a list of dictionaries. Each dictionary contains:
-                * "features": a list of the normalized meta features for a neighbor.
-                * "choice": a integer of the configuration id in the portfolio.
-            - "portfolio": a list of dictionaries, each corresponding to a configuration:
-                * "class": a str of the learner name.
-                * "hyperparameters": a dict of the config. They key "FLAML_sample_size" will be ignored.
-        location: (Optional) A str of the location containing mined portfolio file.
-            Only valid when the portfolio is a str, by default the location is flaml/default.
-
-    Returns:
-        hyperparams: A dict of the hyperparameter configurations.
-        estiamtor_class: A class of the underlying estimator, e.g., lightgbm.LGBMClassifier.
-        X: the preprocessed X.
-        y: the preprocessed y.
-        feature_transformer: a data transformer that can be applied to X_test.
-        label_transformer: a label transformer that can be applied to y_test.
-    """
-    dt = DataTransformer()
-    X, y = dt.fit_transform(X, y, task)
-    if "choose_xgb" == estimator_or_predictor:
-        # choose between xgb_limitdepth and xgboost
-        estimator_or_predictor = suggest_learner(
-            task,
-            X,
-            y,
-            estimator_list=["xgb_limitdepth", "xgboost"],
-            location=location,
-        )
-    config = suggest_config(task, X, y, estimator_or_predictor, location=location, k=1)[
-        0
-    ]
-    estimator = config["class"]
-    model_class = get_estimator_class(task, estimator)
-    hyperparams = config["hyperparameters"]
-    model = model_class(task=task, **hyperparams)
-    if model.estimator_class is None:
-        return hyperparams, model_class, X, y, None, None
-    else:
-        estimator_class = model.estimator_class
-        X = model._preprocess(X)
-        hyperparams = hyperparams and model.params
-
-        class AutoMLTransformer:
-            def transform(self, X):
-                return model._preprocess(dt.transform(X))
-
-        transformer = AutoMLTransformer()
-        return hyperparams, estimator_class, X, y, transformer, dt.label_transformer
+import numpy as np
```

Comment:
Entire file is shown as edited, which makes it hard to review. Make sure EOL (LF vs CRLF) is kept consistent, unless we are actually changing the EOL.

Reply:
Hey. Thanks for the review. Yeah, looks like one of our editors wasn't happy with CRLF. I've reverted the line endings back to DOS